Cakes-Co/style.css

```
@@ -1 +1,400 @@
-/*  Add your styling here */
+/* Add your styling here */
+:root {
+    --bg-color: #e7e5e5;
+    --secondary-color: #f8f8f8;
+    --accent-color: #823a85;
+    --logo-size: 15vh;
+    --nav-color: #120990;
+}
+
+body {
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+}
+/* design using the Mobile First approach */
+.logo-container {
+    display: grid;
+    grid-template-areas: "logo text"; 
+    align-items: center;
+    justify-content: space-between;
+    padding: 1vh 3vw;
+    margin-bottom: 1vh;
+    background-color: var(--bg-color);
+}
+
+.svg-container {
+    width: var(--logo-size);
+    height: auto;
+    grid-area: logo; 
+}
+
+.logo-text {
+    font-weight: bold;
+    grid-area: text; 
+    align-self: end;
+    font-style: italic;
+    color: var(--accent-color);
+    text-align: right; 
+    font-size: 1.9rem;
```

```suggestion
    font-size: var(--font-size--1)
```

Just as you defined colours up front, try defining a type scale and assigning those. In this way you create an organised design system.